neuro_san/coded_tools/website_search/website_search.py

```diff
@@ -17,6 +17,14 @@ class WebsiteSearch(CodedTool):
     def __init__(self):
         self.top_n = 5
 
+    async def async_invoke(self, args: Dict[str, Any], sly_data: Dict[str, Any]) -> Union[Dict[str, Any], str]:
+        """
+        As much as we'd prefer an asynchronous entry point, the code below uses synchronous
+        calls for a simple example. Asynchronous calls make for a more performant server
+        environment.
+        """
+        raise NotImplementedError
```

Comment:
Same as above.

It's possible we should consolidate WebSearch and WebsiteSearch tools as well.
There are slight differences. @Noravee ?